Framework/PythonInterface/plugins/algorithms/SaveReflections.py

```diff
@@ -96,15 +96,16 @@ def PyInit(self):
         self.declareProperty(name="Format",
                              direction=Direction.Input,
                              defaultValue="Fullprof",
-                             validator=StringListValidator(dir(ReflectionFormat)),
+                             validator=StringListValidator(
+                                 [fmt for fmt in dir(ReflectionFormat) if not fmt.startswith('__')]),
```

Comment:
Why not just do

```suggestion
                                 [fmt.name for fmt in ReflectionFormat]),
```

Reply:
Thanks, I didn't know about that - I have committed your suggestion.